### DISCUSSION TITLE
How can I use a template?

### DISCUSSION THREAD

**Discussion by mattmischuk** [PavelLaptev/JSON-to-Figma#31] [Q&A] [answered]
Say I have a mixture of data in one sentence. How would I achieve that? When I populate the fields it looks like it wipes the entire text field despite having inline fields.  eg. `The {dogName} walks to the {locationName}.`

**Comment by PavelLaptev** [comment, marked as answer]
@mattmischuk current version don't have the functionality when you can replace certain words. But I'll bring it back in the next release. I thought nobody use it.
But you can find it in the previous version 3.2 https://github.com/PavelLaptev/JSON-to-Figma/releases/tag/3.2

**Reply by mattmischuk** [reply]
It’s likely a case of non developers not understanding how this type of tooling works. I’d be happy to collaborate with you to make some documentation to cover this.

**Reply by PavelLaptev** [reply]
Cool, thanks!
